cypress/integration/common.js

```diff
@@ -313,6 +314,32 @@ describe('user', () => {
             );
         });
     });
+
+    it('should rate-limit users attempting to login too often', () => {
+        const fakeEmail = faker.internet.exampleEmail();
+        const fakePassword = 'hunter2';
+        const maxAttempts = 10;
+        times(maxAttempts, function(i) {
+            cy.loginUser({
+                username: fakeEmail,
+                password: fakePassword
+            }).then(response => {
+                expect(response.status).to.eq(200);
+            });
+
+            // If we're at the end, run a final one to ensure we're blocked
+            if (i === maxAttempts - 1) {
+                cy.loginUser({
```

Comment:
Could this be called after the `times` block rather than needing to check the index?